development/idp-local/config/authsources.php

```diff
@@ -77,7 +91,7 @@
                 gmdate('YmdHis\Z', strtotime('+6 months')),
             ],
             'mfa' => [
-                'prompt' => 'yes',
+                'prompt' => 'no',
```

Comment:
Why this change (and the other `mfa.prompt` changes from "yes" to "no")? (I presume they are legitimately needed, I'd simply like to learn why, to better understand.)

Reply:
As far as I can tell, the value of `prompt` was a "don't care" before the modules were merged together. In the simplesamlphp-module-profilereview repo (where these test users came from), the mfa module wasn't enabled, so the `mfa.prompt` attribute was ignored. Now that all modules are enabled during test, this needs to be set to `no` so as to not interfere with the `profilereview` feature tests.